package.json

```diff
@@ -134,6 +134,7 @@
   "devDependencies": {
     "gulp-standard": "^8.0.3",
     "gulp-tape": "0.0.9",
+    "sinon": "^1.17.7",
```

Comment:
Doesn't matter, HMRC Prototype Kit is deprecated. They're not prod dependencies so they're going in `devDependencies` as they rightly should.